apps/artist/client/index.coffee

```diff
@@ -1,11 +1,25 @@
-{ ARTIST } = sd = require('sharify').data
+{ ARTIST, IS_PAYOFF, CURRENT_USER } = sd = require('sharify').data
 Backbone = require 'backbone'
 scrollFrame = require 'scroll-frame'
 Artist = require '../../../models/artist.coffee'
 CurrentUser = require '../../../models/current_user.coffee'
 ArtistRouter = require './router.coffee'
+FollowedArtistsRailView = require '../../home/components/followed_artists/view.coffee'
 
 module.exports.init = ->
+  if IS_PAYOFF
+    view = new FollowedArtistsRailView
+      $el: $("#artist-page")
+      user: CURRENT_USER
+      module: 
```

Comment:
These params are (currently) required for the component to render/work, since it's more a homepage thing. We'll probably refactor this.